cmd/chart-repo/sync.go

```diff
@@ -18,10 +18,11 @@ package main
 
 import (
 	"os"
+	"strings"
 
-	"github.com/spf13/cobra"
-	"github.com/sirupsen/logrus"
 	"github.com/kubeapps/common/datastore"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
```

Comment:
this reordering is performed by the GoSublime plugin. Let me know if you want it to be reverted (i prefer that it remains)